napari/utils/transforms/transforms.py

```diff
@@ -429,7 +430,7 @@ def ndim(self) -> int:
         return self._linear_matrix.shape[0]
 
     @property
-    def scale(self) -> np.array:
+    def scale(self) -> npt.NDArray:
```

Comment:
why `NDArray`? As I understand it is to specify dtype https://numpy.org/devdocs/reference/typing.html#numpy.typing.NDArray